pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "pybind11==2.11.1"]
```

Comment:
This is where we specify the pybind11 requirement for building. Instead of a submodule, a version spec can be used here.